bokun.yaml

```diff
@@ -6454,44 +6852,66 @@ definitions:
       - OTHER
   PricingCategoryDto:
     properties:
-      ticketCategory:
-        $ref: '#/definitions/TicketCategoryEnum'
-      ageQualified:
-        type: boolean
-      defaultCategory:
-        type: boolean
-      dependent:
-        type: boolean
-      flags:
-        type: array
-        items:
-          type: string
-      fullTitle:
-        type: string
       id:
         type: integer
         format: int64
-      internalUseOnly:
+        description: 'Specifies ID of the price category.'
+      title:
+        type: string
+        description: 'Specifies title of the price category.'
+      ticketCategory:
+        $ref: '#/definitions/TicketCategoryEnum'
+      occupancy:
+        type: integer
+        format: int32
+        description: 'Specifies how many places for activity availabilities occupies 1 ticket for this price category.'
+      groupSize:
+        type: integer
+        format: int32
+        description: 'Deprecated.'
+      ageQualified:
         type: boolean
-      masterCategoryId:
+        description: 'Specifies whether this price category has age restrictions.'
+      minAge:
         type: integer
-        format: int64
+        format: int32
+        description: 'Specifies lower limit for age restriction for this price category.'
       maxAge:
         type: integer
         format: int32
-      maxDependentSum:
+        description: 'Specifies upper limit for age restriction for this price category.'
+      dependent:
+        type: boolean
+        description: 'Specifies if this price category has booking limits which depend on another price categories in this booking (e.g. max 4 children per 1 adult).'
+      masterCategoryId:
         type: integer
-        format: int32
+        format: int64
+        description: 'Specifies ID of related price category: if dependent == true then master price category id, if dependent == false then child price category id.'
       maxPerMaster:
         type: integer
         format: int32
-      minAge:
+        description: 'Specifies maximum of categories that are set as dependent for one master category. (e.g. 4 children per 1 adult).'
+      sumDependentCategories:
+        type: boolean
+        description: 'Specifies if max sum of dependent categories in one booking is set.'
+      maxDependentSum:
         type: integer
         format: int32
-      sumDependentCategories:
+        description: 'Specifies max sum for dependent categories in one booking.'
+      internalUseOnly:
         type: boolean
-      title:
+        description: 'Specifies if this price category is intended for internal use only.'
+      flags:
+        type: array
+        items:
+          type: string
+        description: 'Not working.'
```

Comment:
remove or fill a bug

Reply:
Recommendation: file as bug (flags for pricing categories not in API response)